goodtablesio/config.py

```diff
@@ -1,13 +1,21 @@
 import os
+import logging
 from dotenv import load_dotenv
 load_dotenv('.env')
 
 
+# General
+
+TABULAR_EXTENSIONS = ['csv', 'xls', 'xlsx', 'ods']
```

Comment:
@roll it would be really good if we can always get this from tabulator, as a property of the package, or a class method on the table class or something. that would mean that we can just always support whatever tabulator declares it can support, and not a hard coded list here.